roles/setup_netobserv_stack/templates/flow-collector.yml.j2

```
@@ -0,0 +1,113 @@
+apiVersion: flows.netobserv.io/v1beta1
+kind: FlowCollector
+metadata:
+  name: "cluster"
+spec:
+  namespace: "netobserv"
+  deploymentModel: DIRECT
+  agent:
+    ebpf:
+      excludeInterfaces:
+        - lo
+      imagePullPolicy: IfNotPresent
+      interfaces: []
+      logLevel: info
+      privileged: {{ setup_netobserv_stack_agent_privileged | default(dsetup_netobserv_stack_agent_privileged) }}
```

Similarly, when you drop the `d` prefix in default values, this becomes
```suggestion
      privileged: {{ setup_netobserv_stack_agent_privileged }}
```

---

done